public/docs/_examples/template-syntax/ts/src/app/app.component.html

```diff
@@ -655,14 +655,12 @@ <h4 id="ngFor-trackBy">*ngFor trackBy</h4>
 <!-- NgSwitch binding -->
 <hr><h2 id="ngSwitch">NgSwitch Binding</h2>
 
-<div>Pick your favorite hero</div>
-<p>
-  <ng-container *ngFor="let h of heroes">
-    <label>
-      <input type="radio" name="heroes" [(ngModel)]="currentHero" [value]="h">{{h.name}}
-    </label>
-  </ng-container>
-</p>
+<p>Pick your favorite hero</p>
+<div>
+  <label *ngFor="let h of heroes">
```

Comment:
It is not. Like for the structural directives page, this is just support code -- i.e., it is not featured on the page.